dendrite-config.yaml

```diff
@@ -68,6 +68,9 @@ global:
   # to other servers and the federation API will not be exposed.
   disable_federation: false
 
+  # Disable presence. Dendrite will not handle presence events.
+  disable_presence: true
```

Comment:
Unless there's a considerable performance hit from having presence enabled, do we want to leave it enabled in our sample config?

Reply:
It's disabled by default :)